src/actions/dbActions.ts

```diff
@@ -168,3 +173,26 @@ export const populateMessageAndPoints = (
     params,
   };
 };
+
+export const _getMessagesAndPoints = async (
+  messageIds: string[],
+  db: USHINBase
+) => {
```

Comment:
We could pass in an additional argument `messagesAndPointsToOmit` (or something like that), which would let ushin-db know which messages and points we already have.